database/generateData.js

```diff
@@ -0,0 +1,64 @@
+const fs = require('fs');
+const faker = require('faker');
+
+const place = {};
+faker.locale = 'en_US';
+//count down from 10m
+const total = 10000000;
+
+const createSidebars = (i) => {
+	if (i === 1) {
+		console.log('record', i, 'created');
+	};
+  place.id = i;
```

Comment:
Might be cleaner to assign them all in a single object block.

Reply:
Agreed, will consider refactoring.